### PR TITLE
chore: azure complete example

### PR DESCRIPTION
**Pull request by karash1989**
### Description of changes

 Complete example for Azure. 

### Checklist

- [X] Title of the pull request follows [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: charts/dial/examples/azure/complete/README.md [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+- [DNS records](https://learn.microsoft.com/en-us/azure/dns/public-dns-overview) configured for Chat
+- [Azure-managed SSL certificates](https://learn.microsoft.com/en-us/azure/app-service/configure-ssl-app-service-certificate) issued for Chat
+- [Application Gateway Ingress Controller](https://github.com/Azure/application-gateway-kubernetes-ingress) installed and configured
+- [Azure Blob storage](https://learn.microsoft.com/en-us/azure/storage/blobs/storage-blobs-overview)
+- [Azure Cache for redis](https://learn.microsoft.com/en-us/azure/azure-cache-for-redis/quickstart-create-redis)
+- [Azure OpenAI](https://learn.microsoft.com/en-us/azure/ai-services/openai/overview)  `gpt-35-turbo` model deployed:
+  - [OpenAI Model Deployment Guide](https://docs.epam-rail.com/Deployment/OpenAI%20Model%20Deployment)
+- [Google Vertex AI](https://cloud.google.com/vertex-ai/?hl=en) `gemini-1.5-pro` model deployed:
+  - [GCP Model Deployment Guide](https://docs.epam-rail.com/Deployment/Vertex%20Model%20Deployment)
+- [Amazon Bedrock](https://docs.aws.amazon.com/bedrock/latest/userguide/what-is-bedrock.html) `anthropic.claude-v1` model deployed:
```
Comment (alexey-ban) [inline charts/dial/examples/azure/complete/README.md:29]: anthropic.claude-v2:1

File: charts/dial/examples/azure/complete/values.yaml [hunk 2]
```diff
@@ -0,0 +1,197 @@
+core:
+  enabled: true
+  serviceAccount:
+    create: true
+    annotations:
+      azure.workload.identity/client-id: "%%AZURE_WORKLOAD_IDENTITY_CLIENT_ID%%"
+      azure.workload.identity/tenant-id: "%%AZURE_TENANT_ID%%"
```
Comment (alexey-ban) [inline charts/dial/examples/azure/complete/values.yaml:7]: Is it required?

File: charts/dial/examples/azure/complete/values.yaml [hunk 3]
```diff
@@ -0,0 +1,197 @@
...
+           "gemini-1.5-pro-preview-0409": {
+            "type": "chat",
+            "displayName": "Gemini 1.5 Pro",
+            "iconUrl": "Gemini-Pro-Vision.svg",
+            "endpoint": "http://dial-vertexai.%%NAMESPACE%%.svc.cluster.local/openai/deployments/gemini-1.5-pro-preview-0409/chat/completions"
+          },
+          "anthropic.claude-v2:1": {
+              "type": "chat",
+              "displayName": "Anthropic (Claude)",
+              "iconUrl": "/anthropic.svg",
```
Comment (alexey-ban) [inline charts/dial/examples/azure/complete/values.yaml:59]: del / in value

File: charts/dial/examples/azure/complete/README.md (hunk 1 above)
Comment (karash1989) [reply charts/dial/examples/azure/complete/README.md]: Updated with anthropic.claude-v2:1

File: charts/dial/examples/azure/complete/values.yaml (hunk 2 above)
Comment (karash1989) [reply charts/dial/examples/azure/complete/values.yaml]: Removed tenant-id as it is an optional value

File: charts/dial/examples/azure/complete/values.yaml (hunk 3 above)
Comment (karash1989) [reply charts/dial/examples/azure/complete/values.yaml]: Removed /

**Pull request merged**